revolv/project/tests/test_models.py

```diff
@@ -11,6 +12,31 @@
 from revolv.project.models import Category, Project
 from revolv.project.tasks import scrape
 
+class ProjectUpdateTest(TestCase):
+    """Tests that check that project updates work with projects"""
+
+    def test_construct(self):
+        project = Project.factories.base.create()
+        update1 = ProjectUpdate(update_text = 'This is update text', project=project)
+        update2 = ProjectUpdate(update_text = 'This is another update', project=project)
+
+        # tests basic construction
+        self.assertEqual('This is update text', update1.update_text)
+        self.assertEqual('This is another update', update2.update_text)
+
+        # tests project relationship
+        self.assertEqual(update1.project_id, update2.project_id)
+        self.assertEqual(update1.project_id, project.id)
+
+        update1.save()
+        update2.save()
+        self.assertEqual(len(ProjectUpdate.objects.all()),2)
+
+    def test_add_update(self):
+        project = Project.factories.base.create()
+        project.add_update('Another sample update')
+        update3 = ProjectUpdate.objects.get(update_text='Another sample update')
+        self.assertEqual(project.id, update3.project_id)
```

Comment:
You should compare `project` and `update3.project` here instead. Also, why is it called update3? why not just update or test_update?